homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -204,6 +204,11 @@ def media_image_url(self):
             self._album_name = await fs_device.get_play_album()
 
             self._source = await fs_device.get_mode()
+            # There appears to be a bug in the API where if the first source on the
+            # list is selected it returns the string "None".
```

Comment:
We should not fix a bug in the library in HA's codebase, please open an PR in the repo of the library.

Reply:
OK I found and fixed the issue in the library and submitted a PR zhelev/python-afsapi#3 if the PR gets accepted how do we get the updated library into HA?

Reply:
Update `manifest.json` with the new version number.